workspaces/announcements/package.json

```diff
@@ -1,50 +1,54 @@
 {
   "name": "@internal/announcements",
   "version": "1.0.0",
+  "private": true,
   "engines": {
-    "node": "18 || 20"
+    "node": "20 || 22"
```

Comment:
The CI currently only runs 18 and 20 so I would probably not change this for now.